Tests/Unit/Filters/ViewData/ChoiceTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\FilterManagerBundle\Tests\Unit\Filters\ViewData;
+
+use ONGR\FilterManagerBundle\Filters\ViewData\Choice;
+
+class ChoiceTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Choice
+     */
+    private $choice;
+
+    /**
+     * Tests setUp method.
+     */
+    protected function setUp()
+    {
+        $this->choice = new Choice();
+    }
+
+    /**
+     * Tests testGetCount method.
+     */
+    public function testGetCount()
+    {
+        $this->choice->setCount(10);
+        $this->assertEquals(10, $this->choice->getCount());
+    }
+
+    /**
+     * Tests testGetLabel method.
+     */
+    public function testGetLabel()
+    {
+        $this->choice->setLabel('test set label');
+        $this->assertEquals('test set label', $this->choice->getLabel());
+    }
+
+    /**
+     * Tests testGetUrlParameters method.
+     */
+    public function testGetUrlParameters()
+    {
+        $this->choice->setUrlParameters([]);
```

Comment:
Here would be better to set some actual parameters instead of empty array. Just to make sure the setter function doesn't modifies the data.